License forbidden (#7871)
Homebrew/brew: docs/Manpage.md

    *Default:* `15`.

  * `HOMEBREW_FORBIDDEN_LICENSES`:

>     Use this environment variable to define a blacklist of space separated licenses and Homebrew will avoid installing the packages with those licenses.

Reviewer: ```suggestion
    Use this environment variable to define a denylist of space separated licenses and Homebrew will refuse to install packages known to have those licenses.
```
